### PR TITLE
[uv.h] document memory functions and loadavg

### PR DESCRIPTION
**Pull request by indutny**
nt

### CODE REVIEW

File: include/uv.h [hunk 1]
```diff
@@ -1063,7 +1063,7 @@ struct uv_fs_event_s {
 };
 
 
-/* Gets load avg */
+/* Gets load avg (in same format as in /proc/loadavg) */
```
Comment (ry) [inline include/uv.h]: can you describe it in a cross platform way?

Comment (indutny) [inline include/uv.h]: Is link to wkipedia article enough? :)

Comment (ry) [inline include/uv.h]: sure

**Comment by ry** [comment]
landed in 5ac040069eed526df7b280e3cdf39d06ca42444a. thanks

**Pull request closed without merging**